### PR TITLE
set up new page to view details of events

### PR DESCRIPTION
**Pull request by Marlin-Noethig**
Established full crud functionality for events. Styling for form not done yet. This is what it looks like: 
Display:
Bildschirmfoto 2022-06-13 um 20 39 59
Add/Edit
Bildschirmfoto 2022-06-13 um 20 40 04

### CODE REVIEW

File: frontend/src/components/budget-overview/BalanceView.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import "./styles/BalanceView.css"
-import {formatMoney} from "../../service/utils/beatifyHelpers";
+import {formatMoney} from "../../service/utils/formatingHelpers";
```
Comment (Boltzmann) [inline frontend/src/components/budget-overview/BalanceView.tsx:2]:
```suggestion
import {formatMoney} from "../../service/utils/formattingHelpers";
```

File: frontend/src/pages/EventDetailsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                            updateEvent={updateEvent}
+                            toggleEnableEdit={toggleEnableEdit}
+                />
+                :
+                <div>
+                    <DisplayEventDetails event={displayedEvent} users={usersWithUserRole}/>
+                    <button onClick={onDelete}>delete</button>
+                    <button onClick={toggleEnableEdit}>edit</button>
+                </div>
+            }
```
Comment (Boltzmann) [inline frontend/src/pages/EventDetailsPage.tsx:52]: Incosistent placement of "?" and ":"?

File: frontend/src/service/api-service/events-api-service.ts [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+const baseUrl: string = "/api/events/"
 
 export const getEvents: (token?: string) => Promise<EventData[]> = (token) =>{
-    return axios.get(baseURl, token
+    return axios.get(baseUrl, token
         ? {headers: {"Authorization": token}}
         : {})
         .then(response => response.data);
+}
+
```
Comment (Boltzmann) [inline frontend/src/service/api-service/events-api-service.ts:12]:
```suggestion
```

File: frontend/src/components/budget-overview/BalanceView.tsx (hunk 1 above)
Comment (Marlin-Noethig) [reply frontend/src/components/budget-overview/BalanceView.tsx:2]: I'll do it in the next Update

File: frontend/src/pages/EventDetailsPage.tsx (hunk 2 above)
Comment (Marlin-Noethig) [reply frontend/src/pages/EventDetailsPage.tsx:52]: I do it quite often like this

**Pull request merged**